docs/install/install.rst

```diff
@@ -0,0 +1,237 @@
+Linux Compiling and Installing
+==============================
+
+This page contains sections as follows:
+
+.. contents::
+    :local:
+
+General Prerequisites
+*********************
+
+Freeciv21 has a number of prerequisites.  Note, that apart from the first prerequisite, the Freeciv21
+configuration process is smart enough to work out whether your system is suitable. If in doubt, just try it.
+
+* An operating system that support Qt.
+
+  Any modern operating system that support Qt is required. As of this writing this is Linux, Microsoft
+  Windows\ |reg| and Apple Mac OS X\ |reg|. On Windows Msys2 is supported.
+
+* A C and C++ 14 compiler.
+
+  Freeciv21 is written in very portable C and C++. Both 32- and 64-bit machines are supported. You cannot
+  use a "K&R C" compiler, or a C++ compiler. The C++ compiler must support C++ 14.
+
+  Development of Freeciv21 is primarily done with :file:`gcc`, the GNU project's excellent C and C++
+  compiler. Microsoft Windows MS Visual C support is under development.
+
+* The :file:`cmake` program.
+
+  Freeciv21 developers generally use :file:`cmake`, the Kitware make program. You can check if you have
+  :file:`cmake` installed on your system by typing the following command. The output should include "Kitware
+  cmake" somewhere and the version should be >=3.12.
+
+.. code-block:: rst
+
+  $ cmake --version
+
+
+* The Ninja cmake build program
+
+
+* :file:`libtool`
+
+  version 2.2 or better
+
+* :file:`libsqlite3`
+
+  http://www.sqlite.org/
+
+  The git version has additional requirements. A release version contains the generated files.

```

Comment:
Not sure what it means but all versions require the same thing.

Reply:
This was in original INSTALL file. I'll update.